examples/nidcpower_source_dc_voltage/_helpers.py

```diff
@@ -231,3 +236,40 @@ def use_simulation_option(default: bool) -> Callable[[F], F]:
         is_flag=True,
         help="Use simulated instruments.",
     )
+
+
+def get_grpc_device_channel(
+    measurement_service: MeasurementService, driver_module: types.ModuleType
+) -> grpc.Channel:
+    """Returns driver specific grpc device channel."""
+    return measurement_service.get_channel(
+        provided_interface=getattr(driver_module, "GRPC_SERVICE_INTERFACE_NAME"),
+        service_class="ni.measurementlink.v1.grpcdeviceserver",
+    )
+
+
+def create_session_management_client(
+    measurement_service: MeasurementService,
+) -> nims.session_management.Client:
+    """Return created session management client."""
+    return nims.session_management.Client(
+        grpc_channel=measurement_service.get_channel(
+            provided_interface=nims.session_management.GRPC_SERVICE_INTERFACE_NAME,
+            service_class=nims.session_management.GRPC_SERVICE_CLASS,
+        )
+    )
+
+
+def get_session_and_channel_string(pin, sessions):
```

Comment:
The function name implies that it returns a tuple containing the session and the channel string, but it actually returns a list of sessions and no channel string.

If you want the function to return one session, it must take the site number as a parameter.

If you want the function to not take the site number, it must return more than one session. Also, rename to `get_sessions_and_channel_strings` and add a list of channel strings.

Reply:
I moved the method addition to a new PR - #276